templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -251,33 +256,73 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <connection host="s3.bwsfs.uni-freiburg.de" port="443" is_secure="True" conn_path="" multipart="True"/>
-            <cache path="/data/jwd02f/s3_object_store_cache" size="10000" />
-            <extra_dir type="job_work" path="/data/jwd02f/main"/>
-            <extra_dir type="temp" path="/data/jwd02f/tmp"/>
+            <cache path="{{ object_store_cache_dir }}" size="10000" cache_updated_data="true" />
+            <extra_dir type="job_work" path="{{ jwd.jwd06.path }}/main"/>
+            <extra_dir type="temp" path="{{ jwd.jwd06.path }}/tmp"/>
+        </backend>
+        <backend type="generic_s3" id="s3_scratch_netapp02" allow_selection="true" weight="0" store_by="uuid" device="scratch" name="Short term storage for method development">
+            <description>S3-based object storage is maintained by the compute center of the University of Freiburg.
+                         Data on this storage, also called scratch storage, is deleted if it is older than two months (60 days), so it is only appropriate for short-term methods development.
+                         The rapid deletion of stored data enables us to provide this storage without a quota. This storage is not backed up.
+                         More information about our storage offerings is documented at [https://galaxyproject.org/eu/storage/](https://galaxyproject.org/eu/storage/).
+            </description>
+            <quota enabled="false" />
+            <badges>
+                <less_stable />
+                <not_backed_up />
+                <short_term>The data stored here is purged after two months (60 days).</short_term>
+            </badges>
+            <auth access_key="{{ s3_netapp01.usegalaxy_eu.access_key }}" secret_key="{{ s3_netapp01.usegalaxy_eu.secret_key }}" />
+            <bucket name="{{ s3_netapp01.usegalaxy_eu.bucket_name }}" use_reduced_redundancy="false" max_chunk_size="250" />
+            <connection host="s3.bwsfs.uni-freiburg.de" port="443" is_secure="True" conn_path="" multipart="True"/>
+            <cache path="{{ object_store_cache_dir }}" size="10000" cache_updated_data="true" />
+            <extra_dir type="job_work" path="{{ jwd.jwd07.path }}/main"/>
+            <extra_dir type="temp" path="{{ jwd.jwd07.path }}/tmp"/>
         </backend>
 
+        <!-- private object store -->
+        <backend type="generic_s3" id="s3_private_netapp01" allow_selection="true" weight="0" store_by="uuid" private="true" name="User-private storage with additional safeguards">
+            <description>S3-based object storage is maintained by the compute center of the University of Freiburg.
+                         Data on this storage cannot be used for public datasets, cannot be shared between users, etc..
+                         All your data in Galaxy is by default only available to you and can not be seen by other users. However, you can always share data, and histories with others.
```

Comment:
The description is confusing in terms of sharing with others and so on.

Reply:
Is it better now?